ui/src/setupProxy.js

```diff
@@ -13,7 +13,21 @@ module.exports = function(app) {
   }
   app.use('/api/login', (req, res, next)=>{
     res.cookie(cookieName, cookieValue,cookieOptions)
-    res.json({status: "success"})
+    res.json({
```

Comment:
This response makes every call to `/api/login` to fail with 401 status code. It needs some logic to check password. Since it is for development purpose only, we can do something simple as checking length of the password to be 12 characters (to match what managed app requires).

Reply:
yes, i have it set this way so everyone can test and see the incorrect but i can add that too

Reply:
done